moto/athena/models.py

```diff
@@ -367,6 +379,25 @@ def stop_query_execution(self, exec_id: str) -> None:
         execution = self.executions[exec_id]
         execution.status = "CANCELLED"
 
+    def create_capacity_reservation(
+        self,
+        name: str,
+        target_dpus:int,
+        tags: List[Dict[str, str]],
+    ) -> None:
+        cr = CapacityReservation(
+            name=name,
+            target_dpus=target_dpus,
+            tags=tags
+        )
+        #TODO: Need some validations
+        self.capacity_reservations[cr.name] = cr
+        return None
+        
+
+    def get_capacity_reservation(self, name: str) -> Optional[CapacityReservation]:
+        return self.capacity_reservations[name] if name in self.capacity_reservations else None
```

Comment:
```suggestion
        return self.capacity_reservations.get(name)
```
Minor nitpick - but this is a bit more readable IMHO